django_app/redbox_app/redbox_core/admin.py

```diff
@@ -82,26 +82,35 @@ class CitationInline(admin.StackedInline):
     extra = 1
 
 
-class ChatMessageAdmin(admin.ModelAdmin):
-    list_display = ["text", "role", "get_user", "chat", "route", "created_at"]
+class ChatMessageAdmin(ExportMixin, admin.ModelAdmin):
+    list_display = ["short_text", "role", "get_user", "chat", "route", "created_at"]
     list_filter = ["role", "route", "chat__user"]
     date_hierarchy = "created_at"
     inlines = [CitationInline]
+    readonly_fields = ["selected_files", "source_files"]
```

Comment:
this makes the admin faster because it only needs to load the actual files associated with the `ChatMessage` and not all possible files that it could be associated to